datasets/acronym_identification/README.md

```diff
@@ -1,35 +1,17 @@
 ---
-annotations_creators:
-- expert-generated
-language_creators:
-- found
-languages:
-- en
-licenses:
-- mit
-multilinguality:
-- monolingual
-size_categories:
-- 10K<n<100K
-source_datasets:
-- original
-task_categories:
-- structure-prediction
-task_ids:
-- structure-prediction-other-acronym-identification
```

Comment:
The tags were removed here. Is this expected ?

Reply:
github mistake